README.md

```diff
@@ -0,0 +1,70 @@
+# Xen vm management (xen_vman)
+
+This role managed xen vms with systemd services.
+
+For each configured vm a xen vm configuration will be auto generated. You can start and stop the vms with systemd.
+Currently iscsi and nfs are completely supported as filesystem backend. For all other backends you need to install the system afterwards manually.
+
+## systemd services
+
+### vm@&lt;organisation&gt;-&lt;host&gt;.service
+Start and stop the given VM. For ex. `systemd start vm@stuvus-web01` will start the _web01_ server from the organisation _stuvus_.
+
+### vm_iscsi@&lt;organisation&gt;-&lt;host&gt;.service
+Does a login or a logout for the iSCSI root disk for the given VM, this is only done, when as backend iSCSI is selected, otherwise systemd will ignore any request silently.
+
+### vm_mount@&lt;organisation&gt;-&lt;host&gt;.service
+Mount or umount the root filesystem from the given VM. For ex. `systemd start vm_mount@stuvus-web02` mounts the root fs from _web01 @stuvus_ into `/mnt/vms/stuvus-web01`. The mount point is automated created and removed.
```

Comment:
*Mounts *umounts. `For ex.` looks silly, use `For example` or `e.g.`

Reply:
*unmounts. `umount` may be the command, but the action is unmounting.